### ISSUE TITLE
Add statistics about number of times a dugga has been marked

### ISSUE THREAD

**Issue by a97marbr** [HGustavs/LenaSYS#2450]
It would be nice to be able to tell how many times a dugga has been marked. This would be useful to

* Tell if a dugga is hard/unclear - the amount of duggas that require more than 2 markings is too hard/unclear
* Rank duggas in terms of how hard they are
* We could perhaps implement something that ask a student to see tutor if dugga has been marked as fail more than 2 times

**Comment by HGustavs** [comment]
The statistics are there in the log file as far as I know, if it is not, then we need to make sure that this information is logged along with everything else.

A nice statistics view that displays this kind of information in a clear manner would be interesting...

**Comment by andras-marki** [comment]
Also it would be useful to determine workload (and thus cost) of the marking.

**Comment by a15andau** [comment]
Is the log file, that is supposed to save the statistics, loglena4.db or is there another log file?

**Comment by a15eriho** [comment]
@a97marbr

**Comment by d15marwi** [comment]
We would like some feedback here, we made two possible solution mock ups and we don’t know which one we should choose.  
1)This one would be applied to the result editor and show the results for each assignment at the bottom as shown in the following picture. Here we could add more rows with different information/data.
![image](https://user-images.githubusercontent.com/37793970/38365678-8566a55a-38dd-11e8-8604-9a0b19d8f4ba.png)

2)The following would be implemented into the section editor. This extra information would be shown on a pc by hovering above the result icon or by pressing on it with a tablet or mobile phone. 
![image](https://user-images.githubusercontent.com/37793970/38365690-955d240c-38dd-11e8-83d6-5d84bf2b8516.png)

**Comment by d15marwi** [comment]
this could be shown in the new statistic overview for students. So instead of showing the stats and deadlines a short overview for a teacher could be shown. We didnt have time to finish this.

**Comment by a17pongo** [comment]
This issue is partly solved by the current state of showing how many times a student has been marked on an assignment. Alot of the Json is already retrieved and could be visualized in different ways.

**Comment by c17johha** [comment]
![image](https://user-images.githubusercontent.com/49199825/58007977-b0957980-7aeb-11e9-942b-f7762e5dea63.png)
does the pie-chart fulfill part of this issue? If so, what remains of the issue?

**Comment by HGustavs** [comment]
Nope, pending is handed in stuff that has not been marked, this is the converse case...

The number of times that a test has been marked including subsequent Fail (U) marks whereas this view only counts the passes or fails one time per section.
